includes/definitions/advafsp150eg-x.yaml

```diff
@@ -0,0 +1,13 @@
+os: advafsp150eg-x
```

Comment:
Ok. Making a product just work or having it structured vendor-centric to follow the overall concept its what missing on my side. I will try to learn from your approach to APC. There you cover also quite a wide range of products with different readings. For now just bin my pull request. Hope you can help here and there by looking at my fork upon request before I place a pull request again. Thx